test/utils.js

```diff
@@ -181,18 +181,22 @@ const utils = module.exports = {
     // CIRRUS_BASE_SHA env variables.
     if (!process.env.FLAKINESS_DASHBOARD_PASSWORD || process.env.CIRRUS_BASE_SHA)
       return;
-    const sha = process.env.FLAKINESS_DASHBOARD_BUILD_SHA;
+    const {sha, timestamp} = await FlakinessDashboard.getCommitDetails(__dirname, 'HEAD');
     const dashboard = new FlakinessDashboard({
-      dashboardName: process.env.FLAKINESS_DASHBOARD_NAME,
+      commit: {
+        sha,
+        timestamp,
+        url: `https://github.com/GoogleChrome/puppeteer/commit/${sha}`,
```

Comment:
is baking in the repo URL necessary here? The commit hash should be enough.

Reply:
Don't want to rely on Github URL schemes too much in flakiness viewer. This comes for free since this is per-build only (and there are not many builds)

Reply:
What happens if we ever move the repo?